### PR TITLE
Changes required for test_crm_fdb_entry to support Cisco-8000 platform

### PR DESCRIPTION
**Pull request by svelamal**
### Description of PR

- For Cisco-8000 devices, for 1 fdb entry added, the "crm_stats_fdb_entry_used" counter increases and "crm_stats_fdb_entry_available" counter decreases by more than 1. 
- This is due to the CEM being a shared resource and the availability being calculated statistically. The results will depend on the current state of the CEM/everything else that is stored. 

### Type of change
- [ ]  Test case(new/improvement)

#### What is the motivation for this PR?
The existing script adds 1 fdb entry in fdbconfig.json and checks for counters crm_stats_fdb_entry_used to increment by 1 and crm_stats_fdb_entry_available to decrement by exactly 1. Because there is no 1:1 correlation between fdb entries and used/available crm counters in Cisco platforms, this testcase fails

#### How did you verify/test it?
Verified on cisco-8000 platform

#### Any platform specific information?
cisco-8000

### CODE REVIEW

**Comment by sachinv-msft** [comment]
@rlhui any recommendations on who should be reviewing this PR?

**Comment by sachinv-msft** [comment]
@yxieca any recommendations on who should be looking into this PR?

**Comment by balajib-cisco** [comment]
> If that is the case, how we can estimate how many fdb entries are available? How the monitor should work and alert if the fdb table reaches the threshold?

Though hardware FDB counter is statistical-based, It is still fairly accurate with +/- 1 entry tolerance i.e.) inaccuracy of (1/622417*100). The problem is, the test script searching for exact count match each one increase or decrease. Once in a while, it is not going to have difference of 1 entry out of total 622417  entries due to hash function involved in hardware table offset calculation.  In nutshell, it is still more than good enough accuracy to use for CRM monitor which generates the syslog threshold messages based on percentage.

**Comment by kevinskwang** [comment]
> 
Ok, make sense to me. Could you add some comments on that code?

**Comment by svelamal** [comment]
> > 
> 
> Ok, make sense to me. Could you add some comments on that code?

Added

File: tests/crm/test_crm.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import tempfile
 
 from jinja2 import Template
+from tests.common.cisco_data import is_cisco_device
```
[the remark this answers is not in the document]
Comment (balajib-cisco) [reply tests/crm/test_crm.py:12]: It is there in tests/common/cisco_data.py

**Review by kevinskwang — APPROVED** [review]
LGTM

File: tests/crm/test_crm.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (svelamal) [reply tests/crm/test_crm.py:12]: Please refer https://github.com/Azure/sonic-mgmt/blob/master/tests/common/cisco_data.py

**Pull request merged**